src/anemoi/training/train/forecaster.py

```diff
@@ -251,6 +251,8 @@ def training_weights_for_imputed_variables(
                     loss_weights_mask = pre_processor.transform_loss_mask(loss_weights_mask)
             # update scaler with loss_weights_mask retrieved from preprocessors
             self.loss.update_scalar(scalar=loss_weights_mask.cpu(), name="loss_weights_mask")
+
+        self.scalars["loss_weights_mask"] = ((-2, -1), loss_weights_mask.cpu())
```

Comment:
This breaks if loss_weights_mask is not mentioned in the training-config file (which is the default). `loss_weights_mask` has to be defined before the if-statement.

Reply:
Fixed in [4f322fa](https://github.com/ecmwf/anemoi-training/pull/202/commits/4f322fa12e90df6f3b6f6666a9b50033ee841d97)